### ISSUE TITLE
`train_online.py` time

### ISSUE THREAD

**Issue by bkj** [kmaninis/OSVOS-PyTorch#17]
Using the default settings, roughly how long should `train_online.py` take to train?  I'm seeing ~1 hour on a TITANX -- is that in the right neighborhood?  (The paper makes it sound like the finetuning runs for < 1 minute.)

~ Ben

**Comment by kmaninis** [comment]
Hi Ben, all times reported in the paper, for OSVOS and competing methods, are per frame. So that would be the right neighbourhood. You can train for less in the cost of accuracy, but in our experience you need more than 5 minutes to get something meaningful.

**Comment by bkj** [comment]
OK -- doesn't `train_online.py` just train on one image though?  I may be misunderstanding exactly what's going on.

**Comment by kmaninis** [comment]
Yes. 1 image and its augmentations.

**Comment by bkj** [comment]
I don't think I understand.  

AFAICT: running
```
python train_online.py
```
calls
```
seq_name = 'blackswan'
db_train = db.DAVIS2016(train=True, db_root_dir=db_root_dir, transform=composed_transforms, seq_name=seq_name)
```
which is a dataset w/ one image (`JPEGImages/480p/blackswan/00000.jpg `).   Then it trains for 10K iterations on that single image, which takes ~1 hour.  

So doesn't this mean that, using the default settings, finetuning takes 1 hour for a single frame?

~ Ben

**Comment by kmaninis** [comment]
You are right. After fine-tuning, the rest of the images are used for scoring only which takes roughly 100ms per image. The timings reported would be (fine-tuning + scoring)/n_frames.

**Comment by bkj** [comment]
OK -- understood, though the metric is a little unexpected.  Might be worthwhile to add expected runtime information to the docs for clarity.

Thanks for your quick help!
